lib/view_component/base.rb

```diff
@@ -16,6 +16,8 @@ class Base < ActionView::Base
     class_attribute :content_areas
     self.content_areas = [] # class_attribute:default doesn't work until Rails 5.2
 
+    mattr_accessor :render_monkey_patch_enabled, instance_writer: false
```

Comment:
Might we group this with the other `mattr_accessor` in this file?

And can we set a default value to true?

Reply:
Good idea! Done in 80f80090b6cae95a95af2407d0854f490808d7de